packages/server/fileStorage/S3Manager.ts

```diff
@@ -1,52 +1,60 @@
-import aws from 'aws-sdk'
+import {HeadObjectCommand, PutObjectCommand, S3Client} from '@aws-sdk/client-s3'
 import mime from 'mime-types'
-import {APP_CDN_USER_ASSET_SUBDIR} from 'parabol-client/utils/constants'
-import protocolRelativeUrl from 'parabol-server/utils/protocolRelativeUrl'
 import path from 'path'
+import url from 'url'
 import FileStoreManager from './FileStoreManager'
 
 export default class S3Manager extends FileStoreManager {
-  private baseUrl: {hostname: string; pathname: string}
+  // e.g. development, production
+  private envSubDir: string
+  // e.g. action-files.parabol.co
   private bucket: string
-  private s3: AWS.S3
+  private s3: S3Client
   constructor() {
     super()
-    const {CDN_BASE_URL, AWS_S3_BUCKET} = process.env
+    const {CDN_BASE_URL, AWS_S3_BUCKET, AWS_REGION} = process.env
```

Comment:
`AWS_REGION` is already present in all the env configs, you just added the usage here, right?